Jenkinsfile

```diff
@@ -230,5 +230,17 @@ pipeline{
         junit '**/build/**/TEST*.xml'
       }
     }
+    stage('regressions-10.0') {
+          when {
+            allOf {
+              branch 'develop'
+              expression {return params.regressions}
+            }
+          }
+          steps {
+            runAllTests('Lates', '10.0', false)
```

Comment:
Looks like a typo? 

Can you also add a regression for "Latest" "11.0"?

Reply:
Thank you for catching that. Sure, will do